fury/primitive.py

```diff
@@ -851,3 +851,235 @@ def prim_cylinder(radius=0.5, height=1, sectors=36, capped=True):
         triangles = (np.array(triangles).reshape(2 * sectors, 3))
 
     return vertices, triangles
+
+
+def prim_para_mobius_strip():
+    """Return vertices and triangle for a Möbius strip
+
+    Returns
+    -------
+    vertices: ndarray
+        vertices coords that compose our Möbius strip
+    triangles: ndarray
+        triangles that compose our Möbius strip
+    """
+
+    npoints = 100
```

Comment:
can you put it as a function parameter with `100`as default value

Reply:
done